### PR TITLE
Create baremtal hosts

### PR DESCRIPTION
**Pull request by IsaacsofferYU**

### CODE REVIEW

File: terraform-environments/baremetal-hosts.tf [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# data "template_file" "user_data" {
+#   template = file("../host_user_data.sh")
+# }
+
+data "template_file" "user_data" {
+  template = <<-EOT
```
Comment (thehcma) [inline terraform-environments/baremetal-hosts.tf:6]: ideally you should factor this out into a separate script (which we can then lint and then validate - and here we would just be calling it

**Review by thehcma — CHANGES_REQUESTED** [review]

File: host_user_data.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (ygins) [inline host_user_data.sh:1]: any user data?

File: terraform-environments/instance.tf [hunk 3]
```diff
@@ -2,7 +2,9 @@ resource "aws_security_group" "security" {
   name        = "ec2_security2"
```
Comment (ygins) [inline terraform-environments/instance.tf:2]: This file should prob be renamed. instance could be anything

File: host_user_data.sh (hunk 2 above)
Comment (ygins) [reply host_user_data.sh:1]: try useradd?

**Review by ygins — CHANGES_REQUESTED** [review]
Left a couple comments. I would rename instance, and if we're using the user file keep it if not get rid of it. We prpobably want to have users

File: terraform-environments/instance.tf (hunk 3 above)
Comment (IsaacsofferYU) [reply terraform-environments/instance.tf:2]: its a lot more used by tani and ari - i just created the outline for it, and i believe they renamed it in their branch

**Review by IsaacsofferYU — COMMENTED** [review]
I have fixed your issues. the blank file is being used in this next stage, but i guess it made its way into this commit

**Review by ygins — APPROVED** [review]
Not too familiar with terraform, but aside from the comments I made before, looks good.

**Pull request closed without merging**